test/test_console_tool.py

```diff
@@ -685,7 +685,7 @@ def test_get_account_info(self):
         self._authorize_account()
         expected_stdout = '''
         {{
-            "accountAuthToken": "auth_token_0",
+            "accountAuthToken": "auth_token_1",
             "accountId": "{account_id}",
```

Comment:
To be honest I dont know why the auth_token numbers were off, but my unit tests fail without this change.
If you know something I don't let me know, I didn't dig too deep to figure out why.

Reply:
Apparently these are wrong this time.. will revert soon once my pre-commit.sh works again.